src/clj/puppetlabs/services/jruby/jruby_schemas.clj

```diff
@@ -99,19 +106,22 @@
                      (nil? (schema/check JRubyInstanceState @%)))
                'JRubyInstanceState))
 
+(def JRubyInstanceSchema
+  {:pool pool-queue-type
+   :id schema/Int
+   :max-requests schema/Int
+   :flush-instance-fn IFn
+   :state JRubyInstanceStateContainer
+   :scripting-container ScriptingContainer
+   schema/Keyword schema/Any})
```

Comment:
The schema check for the record is too strict and doesn't tolerate any additional keys associated onto the record. I tried doing something like `(assoc JRubyInstance schema/Keyword schema/Any)` and `(merge {schema/Keyword schema/Any})` but no incantation of anything l tried worked, unfortunately. Unless anyone has any other ideas, I'll add a comment.